### PR TITLE
Advanced & Syndicate Magboots Provide NOSLIP_ICE when active

### PR DESCRIPTION
**Pull request by VaelophisNyx**
# Document the changes in your pull request

See: Title

# Changelog

:cl:  
tweak: Advanced Magboots & Syndicate Magboots provide Noslip on Ice
/:cl:

### CODE REVIEW

File: code/modules/clothing/shoes/magboots.dm [hunk 1]
```diff
@@ -53,12 +53,24 @@
 	slowdown_active = SHOES_SLOWDOWN
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 
+/obj/item/clothing/shoes/magboots/advance/attack_self(mob/user)
+	if(magpulse)
```
Comment (Bibby0110) [inline code/modules/clothing/shoes/magboots.dm]: never turns off?

File: code/modules/clothing/shoes/magboots.dm [hunk 2]
```diff
@@ -53,12 +53,24 @@
...
+		clothing_flags &= ~NOSLIP | NOSLIP_ICE
+		slowdown = SHOES_SLOWDOWN
```
Comment (Bibby0110) [inline code/modules/clothing/shoes/magboots.dm]: unneeded

File: code/modules/clothing/shoes/magboots.dm [hunk 3]
```diff
@@ -53,12 +53,24 @@
 	slowdown_active = SHOES_SLOWDOWN
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 
+/obj/item/clothing/shoes/magboots/advance/attack_self(mob/user)
+	if(magpulse)
+		clothing_flags &= ~NOSLIP | NOSLIP_ICE
```
Comment (Bibby0110) [inline code/modules/clothing/shoes/magboots.dm]:
```suggestion
		clothing_flags &= ~NOSLIP | ~NOSLIP_ICE
```
i think

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

File: code/modules/clothing/shoes/magboots.dm (hunk 1 above)
Comment (VaelophisNyx) [reply code/modules/clothing/shoes/magboots.dm]: Pretty sure it should be handled fine since it's a subtype?

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**